service-jehlomat/src/main/kotlin/api/VerificationApi.kt

```diff
@@ -2,29 +2,36 @@ package api
 
 import io.ktor.application.*
 import io.ktor.http.*
-import io.ktor.request.*
 import io.ktor.response.*
 import io.ktor.routing.*
-import model.*
 import services.DatabaseService
-import services.Mailer
 
 
 fun Route.verificationApi(database: DatabaseService): Route {
-    return route("/") {
-        get("/{id}") {
-            // TODO: check if values satisfy condition
-            // TODO: check if adminitrator is logged user
+    return get {
+        val organizationId = call.parameters["orgId"]?.toIntOrNull()
+        val userId = call.parameters["userId"]?.toIntOrNull()
 
-            val id = call.parameters["id"]?.toIntOrNull()
-            val currentOrganization = id?.let { it1 -> database.selectOrganizationById(it1) }
+        if (organizationId != null) {
```

Comment:
The organization verification is quite unsecure. It's easy to find out the organization id and simulate the super admin confirmation. I would recommend to implement some method to prove that the endpoint was called by the super admin. 

For example generate and store a random hash during the organization registration. The hash would be send with the id in the confirmation email and then checked against the stored value during the verification.

The same system could be implemented for the user verification, too. But that is not so dangerous as creating an organization by anybody.

Reply:
I thought we will use JWT token to verify super admin. The token will not be sufficient ?

Reply:
You are right, that's another way how to do it, I'm just used to have these email links working without authentication.